Mimir.Worker/StateDocumentConverter/ArenaParticipantDocumentConverter.cs

```diff
@@ -0,0 +1,31 @@
+using Lib9c.Models.Arena;
+using Lib9c.Models.States;
+using Mimir.MongoDB.Bson;
+
+namespace Mimir.Worker.StateDocumentConverter;
+
+public class ArenaParticipantDocumentConverter : IStateDocumentConverter
+{
+    [Obsolete("Use ConvertToDocument(AddressStatePair, int, int, SimplifiedAvatarState) instead.")]
+    public MimirBsonDocument ConvertToDocument(AddressStatePair context)
+    {
+        throw new InvalidOperationException(
```

Comment:
How about using [`NotImplementedException`](https://learn.microsoft.com/en-us/dotnet/api/system.notimplementedexception?view=net-8.0)

Reply:
I wanted to make it clearer that it wasn't that this method wasn't implemented, but that a different method should be used, so I threw an exception for invalid behavior.